src/app/components/normal-component/pick-game/pick-game.component.ts

```diff
@@ -138,6 +141,8 @@ class GameDescription {
 
     public static readonly KAMISADO: Localized = () => $localize`Your goal is simple: reach the last line. But the piece you move depends on your opponent's last move!`;
 
+    public static readonly LASCA: Localized = () => $localize`Close to checkers, capture opponent pieces, free your own, and immobilize you opponent to win the game!`;
```

Comment:
Similar to checkers? Close pour moi est plus une question de distance que de similarité